README.md

```diff
@@ -30,3 +30,11 @@ To test if a translation is working, set the LANG environment variable to desire
 
 `LANG=pl_PL.utf-8 qubes-update-gui`
 
+To push translations to transifex (you will need your own transifex API token):
+
+`tx push --source`
+`tx push --translation`
```

Comment:
Is push --translation really needed? I think it's risky (could roll back translation to a previous version).